src/main/java/hudson/plugins/analysis/core/HealthAwareRecorder.java

```diff
@@ -237,6 +247,10 @@ public HealthAwareRecorder(final String healthy, final String unHealthy,
 
     // CHECKSTYLE:ON
 
+    public HealthAwareRecorder(String pluginName) {
+        this.pluginName = "[" + pluginName + "] ";
+    }
+
```

Comment:
Why did you replace the constructor with setters? This removes the immutabilty state from all fields!